docs_source_files/content/selenium_installation/installing_selenium_libraries.es.md

```diff
@@ -3,18 +3,14 @@ title: "Instalación de las librerías Selenium"
 weight: 1
 ---
 
-{{% notice info %}}
-<i class="fas fa-language"></i> Page being translated from 
-English to Spanish. Do you speak Spanish? Help us to translate
-it by sending us pull requests!
-{{% /notice %}}
 
 First you need to install the Selenium bindings for your automation project.
```

Comment:
There is still some text in English, can you please remove it?

Reply:
Was remove some paragraphs that still remained in English